### PR TITLE
create basic flow

### PR DESCRIPTION
**Pull request by Harmoney-TerryFeng**
1. add basic flow
2. add after step hook to complete user task 

Need to setup 'iamAdminPassword' in env variable

### CODE REVIEW

File: features/step_definitions/IAM.steps.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import {User} from "../../support/components/user";
+
+Given('The IAM is up and running', async function () {
+    const response = await User.iamAdmin(this.servers.iam);
+    this.adminToken = response.body.access_token;
+});
+
+When('I sign up with email {string}', async function (email: string) {
+    const timeStamp = Date.now();
+    email = email.replace('@', `+${timeStamp}@`);
```
Comment (ShawnXia) [inline features/step_definitions/IAM.steps.ts:14]: If we add the random string here, we need to highlight it in the console to allow the user to get the final user name.

File: features/borrowerFunnel.feature [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Feature: As a tester, I want to test the borrower funnel is working as expected
+
+  Scenario: AU Bete borrower flow
+    Given The IAM is up and running
+    When I sign up with email "svc.qauser@harmoney.co.nz"
+    Then user sign up success
+    When I complete page username with name 'random'
```
Comment (ShawnXia) [inline features/borrowerFunnel.feature:7]: This step doesn't work, please have a try from your end.

**Review by ShawnXia — COMMENTED** [review]
Nice work! Please see the comment in the code.

BTW, we need to add a few more features:

- Retry
- ENV definition
- README.md

As your PR has covered my PR, I will close my PR directly.

File: features/step_definitions/IAM.steps.ts (hunk 1 above)
Comment (Harmoney-TerryFeng) [reply features/step_definitions/IAM.steps.ts:14]: Yep, already print the email in another place, but will move the log right after the user gets created.

File: features/borrowerFunnel.feature (hunk 2 above)
Comment (Harmoney-TerryFeng) [reply features/borrowerFunnel.feature:7]: Yes, the api request has been changed lately, need to update again

Comment (ShawnXia) [reply features/borrowerFunnel.feature:7]: 😂

**Review by ShawnXia — APPROVED** [review]

**Pull request merged**